src/explore-education-statistics-common/src/modules/methodology/components/MethodologySectionBlocks.tsx

```diff
@@ -2,16 +2,26 @@ import InsetText from '@common/components/InsetText';
 import ContentBlockRenderer from '@common/modules/find-statistics/components/ContentBlockRenderer';
 import { ContentBlock } from '@common/services/types/blocks';
 import React from 'react';
+import useMethodologyImageAttributeTransformer from '@common/modules/methodology/hooks/useMethodologyImageAttributeTransformer';
 
 interface Props {
   blocks: ContentBlock[];
+  methodologyId: string;
 }
 
-const MethodologySectionBlocks = ({ blocks }: Props) => {
+const MethodologySectionBlocks = ({ blocks, methodologyId }: Props) => {
+  const transformImageAttributes = useMethodologyImageAttributeTransformer({
+    methodologyId,
+    rootUrl: process.env.CONTENT_API_BASE_URL.replace('/api', ''),
```

Comment:
Is this correct if its in common, as just checking this wouldn't be used by the Admin anywhere where the Content API  won't exist.

Reply:
Created ticket EES-1999 to address this.